### ISSUE TITLE
您好，您在KRVQA数据集上微调有出现权重维度不匹配的问题嘛？类似这样的RuntimeError: Error(s) in loading state_dict for KgPreModel:         size mismatch for tail_decode.weight: copying a param with shape torch.Size([146348, 300]) from checkpoint, the shape in current model is torch.Size([22875, 300]).> > Hello, is the result of your pre-training on VQAV2 normal? There's something wrong with my accuracy. It's all 0

### ISSUE THREAD

**Issue by wangfengjuan** [AndersonStra/MuKEA#7]
> > Hello, is the result of your pre-training on VQAV2 normal? There's something wrong with my accuracy. It's all 0

You need to set _--dataset okvqa_ and it will still pre-train on vqav2. _--dataset vqav2_ is just a test.

_Originally posted by @zhongfansun in https://github.com/AndersonStra/MuKEA/issues/4#issuecomment-1107992696_

**Comment by zhongfansun** [comment]
你调用了OKVQA上训练的模型，此时应当调用在KRVQA上预训练的模型

**Comment by BackstreetK** [comment]
你好
请问  你后面是咋解决的？  我在OKVQA上微调也遇到了这个问题？

**Comment by ProudZhangXiaolong** [comment]
> 你好 请问 你后面是咋解决的？ 我在OKVQA上微调也遇到了这个问题？

请问，你解决了吗？
